### PR TITLE
Disable transfer/conference in case 0 agents available and show hover message

### PR DESCRIPTION
**Pull request by mharcar**

### CODE REVIEW

File: src/Agent/Widget/ConferenceAgent.vue [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <template>
 <b-dropdown text="Agent" variant="outline-primary">
-  <b-dropdown-item v-for="agent in this.$agent.vm.transfer_agents" :key="agent.id" @click="onUpdate(agent.id)">{{ agent.login }}</b-dropdown-item>
+  <div v-if="this.$agent.vm.transfer_agents.length > 0">
```
Comment (jamhed) [inline src/Agent/Widget/ConferenceAgent.vue]: Imo it's better not to display the button at all.

Comment (mharcar) [reply src/Agent/Widget/ConferenceAgent.vue]: I did it as was specified in reach3 issue 44

Comment (jamhed) [reply src/Agent/Widget/ConferenceAgent.vue]: These issues aren't final nor strict demands, but things to think about and to discuss.

Comment (jamhed) [reply src/Agent/Widget/ConferenceAgent.vue]: So, as discussed with @jbrowne3, the button should be disabled if there are no agents and display a popover with explanations on mouse hover.

Comment (mharcar) [reply src/Agent/Widget/ConferenceAgent.vue]: done

**Pull request merged**